openedx/adg/lms/certificates_override/helpers.py

```diff
@@ -0,0 +1,81 @@
+"""
+Helpers for certificates
+"""
+from django.utils import translation
+from django.utils.translation import gettext as _
+from opaque_keys.edx.keys import CourseKey
+
+from common.djangoapps.util.date_utils import strftime_localized
+from lms.djangoapps.certificates.models import CertificateStatuses, GeneratedCertificate
+from lms.djangoapps.certificates.views.webview import _get_user_certificate
+from lms.djangoapps.courseware.courses import get_course_by_id
+from openedx.adg.lms.certificates_override.constants import ARABIC_LANGUAGE_CODE
+from openedx.core.djangoapps.certificates.api import display_date_for_certificate
+
+
+def get_arabic_certificate_extra_context(request, course_id):
+    """
+    Compute `date_issued` and `gender` for the Arabic Course Certificates.
+
+    This method is used directly from certificate template. This is done to avoid core changes and customise the
+    Arabic certificates. This method customised formatted date issued of certificate and gender to the user
+    to display custom messages.
```

Comment:
Please go through the third sentence, it needs some reformatting, it's difficult to comprehend